dashboard/src/components/hackathon/HackathonTeamMember.vue

```diff
@@ -186,4 +201,18 @@ const removeTeamMember = (member) => {
   })
   team.fetch()
 }
+const fetchTeamMemberCount = () => {
+  createResource({
+    url: 'fossunited.api.hackathon.get_count_team_members_and_max_count',
+    makeParams() {
+      return {
+        hackathon: props.team.data.hackathon,
+        team: props.team.data.name,
+      }
+    },
+    onSuccess(data) {
+      teamMemberCount.value = data
+    },
+  }).fetch()
+}
```

Comment:
function inside function

why not just

```
const fetchTeamMemberCount = createResource({
 ....
})
```

and do a fetchTeamMemberCount.fetch() in onMounted?